docs/user/graph/getting-started.asciidoc

```diff
@@ -2,30 +2,30 @@
 [[graph-getting-started]]
 == Using Graph
 
-Graph is automatically enabled in {es} and {kib}.
+You must index data into {es} before you can create a graph.  
+<<index-patterns, Learn how>> or get started with a <<add-sample-data, sample data set>>.
 
+[float]
 [[exploring-connections]]
-To start exploring connections in your data:
+=== Graph connections in your data
 
-. From the side navigation, open the graph explorer.
-
-. Select an index pattern to specify what indices you want to explore.
+. From the side navigation, open *Graph*.
 +
-For example, if you are indexing log data with Logstash, you could select the
-`logstash-*` index pattern to visualize connections within the log entries.
+If this is your first graph, follow the prompts to create it.  
```

Comment:
Maybe put this in bullets?  For example:

- If this is your first graph, follow the prompts.
- For subsequent graphs, click New.